tomviz/CMakeLists.txt

```diff
@@ -236,6 +236,7 @@ set(exec_sources
 
 set(python_files
   AddPoissonNoise.py
+  Auto_Correlation.py
```

Comment:
It seems that all other operator file names are using CamelCase, should we be consistent? i.e. AutoCorrelation.py.

Reply:
Yes, we should, good catch

Reply:
I wasn't aware of the formatting or clangformat when I submitted this branch. Thank you for informing me.